src/index.html

```diff
@@ -5,8 +5,239 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="Moyo logo" class="header__image">
+      </a>
+      <nav class="nav">
+        <button class="nav__button">&equiv;</button>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active">Apple</a>
```

Comment:
так як зараз, тести не будуть проходити
```suggestion
            <a href="#" class="nav__link nav__link--is-active">Apple</a>
```